src/Cairo.jl

```diff
@@ -30,14 +30,23 @@ end
 
 include("constants.jl")
 
+# check library version, error message if too low
+cairo_version = ccall((:cairo_version,Cairo._jl_libcairo),Int32,())
+
+if cairo_version < 11200
+    error("cairo version " * string(cairo_version) * ", less then 1.12.00")
```

Comment:
this should be done as a bindeps validation check, or make the extra features disabled if the library is too old